hasura/graphql-engine
# display untrackable functions on schema page (close #2249)

### Description

- display non trackable functions list on schema page (collapsed by default)
  - show trackable functions requirements msg at start
  - show `View` btn which redirects to SQL page with fn definition pre-filled

- display untracked functions section all the time (collapsed by default if no trackable fns)
  - made sense as non trackable fns section wil almost always be shown. might help with feature discoverability as well.
  - show trackable functions requirements msg if no trackable fns found

### Affected components 
- Console

### Related Issues
#2249 

### Solution and Design


### Steps to test and verify

- check UI if no untracked fns vs if untracked fns present
- check `View` UX for non trackable fns
- check if the messaging is clear

### Limitations, known bugs & workarounds
